pkg/operator/controller/ingress/deployment.go

```diff
@@ -1678,3 +1669,11 @@ func GetMIMETypes(mimeTypes []operatorv1.CompressionMIMEType) []string {
 
 	return mimes
 }
+
+func calcReloadIntervalFromInput(interval *metav1.Duration) string {
+	reloadInterval := 5 * time.Second
+	if interval != nil && interval.Duration >= 1*time.Second && interval.Duration <= 720*time.Hour {
+		reloadInterval = interval.Duration
+	}
```

Comment:
I figured values outside of the allowed range would be capped, as the operator does for other timeouts, rather than ignored entirely:
```suggestion
	const (
		defaultInterval = 5 * time.Second
		minInterval = 1 * time.Second
		maxInterval = 720 * time.Hour
	)
	reloadInterval = defaultInterval
	if interval != nil {
		if interval.Duration < minInterval {
			reloadInterval = minInterval
		} else if [etc.]
	}
```
Does that make sense?